### DISCUSSION TITLE
so-allow not working ( we cannot access to security onion console)

### DISCUSSION THREAD

**Discussion by Chizan-Dolrocks** [Security-Onion-Solutions/securityonion#8482] [Unsupported Versions]
Hello,
we have just installed security onion (Latest version: 2.3.140) in distributed mode, we start the manager's installation. once the installation of the manager has been done, we made the command "so-allow" with the option Analyst -a, we assign it the IP address of the analyst network (for example 192.168.1.0/24).
After that, we can't access the security onion console from the analyst network !!!

N.B: At first we  deactivated the "firewalld" of the system, we can access to the security onion console from everywhere.  
When we looked at the status of the "firewalld" it showed us errors in the following capture
![so-erreur](https://user-images.githubusercontent.com/84504744/183354304-a505b2b0-692c-4bbf-b020-b5b0f1d3893a.png)

Thanks in advance

**Comment by dougburks** [comment]
Looking at the screenshot, those appear to be warnings rather than errors.

What is the output of the following?
```
sudo so-allow-view 
```

**Reply by Chizan-Dolrocks** [reply]
The so-allow-view command has this output:

<img width="442" alt="so-erreur3" src="https://user-images.githubusercontent.com/84504744/183644381-9c2a4682-4c1f-4c79-9efd-2ea49feca8b0.png">

even the workstations that are in this network can't access to security onion console when the system's internal firewall (firewall) is active. but once deactivated we can access security onion console from anywhere.

**Comment by mnasec** [comment]
If you using any proxy server, then implement a proxy exception in your browser for the master node.

Von meinem iPhone gesendet
